lab-raziyeh/app/entry.js

```diff
@@ -0,0 +1,47 @@
+'use strict';
+
+// requires scss
+require('./scss/main.scss');
+
+// non angular npm modules
+const path = require('path');
+//const camelcase = require('camelcase');
+const pascalcase = require('pascalcase');
+
+// angular
+const angular = require('angular');
+// require 3rd party angular modules
+const uiRouter = require('angular-ui-router');
+
+// create app module
+const profile = angular.module('profile', [uiRouter]);
+
+let context = require.context('./config/', true, /\.js$/);
+context.keys().forEach(key => {
+  profile.config(context(key));
+});
+
+// load view controllers in ./view
+context = require.context('./view/', true, /\.js$/);
+context.keys().forEach( key => {
+  let name = pascalcase(path.basename(key, '.js'));
+  profile.controller(name, context(key));
+});
+
+// load services 
```

Comment:
@Raziyehbazargan best to remove these comment blocks from `master` to keep it production ready